acceptance_modelisation/base_acceptance_map_creator.py

```diff
@@ -263,12 +263,14 @@ def _transform_exclusion_region_to_camera_frame(self, pointing_altaz: AltAz) ->
                 width_coordinate = center_coordinate.directional_offset_by(region.angle, region.width)
                 width_coordinate_altaz = width_coordinate.transform_to(pointing_altaz)
                 width_coordinate_camera_frame = width_coordinate_altaz.transform_to(camera_frame)
-                angle_camera_frame = center_coordinate_camera_frame.position_angle(width_coordinate_camera_frame).to(u.deg)[0]
-                center_coordinate_camera_frame_arb = SkyCoord(ra=center_coordinate_camera_frame.lon[0],
-                                                              dec=center_coordinate_camera_frame.lat[0])
+                center_coordinate_camera_frame_arb = SkyCoord(ra=-center_coordinate_camera_frame.lat[0],
```

Comment:
Why the new names? (with _arb)

Reply:
It was the existing name, I just changed the SkyCoord values to apply rotoreflexion

Here are the before/after plots obtained on a model of a gaussian background with the stack method, with the circle and the ellipse

![Capture d’écran 2024-11-19 à 17 25 20](https://github.com/user-attachments/assets/88347525-0951-4e30-81bd-935e8f746bfb)